.github/workflows/publish-benchmark.yaml

```diff
@@ -0,0 +1,31 @@
+name: Trigger Site Rebuild on a CRON Schedule
+
+on:
+  schedule:
+    # Runs "at minute 55 past every hour" (see https://crontab.guru)
+    - cron: '0 2 * * *'
```

Comment:
Is GMT used?

Reply:
Yes, I've updated the comment to reflect this + the correct time